configmaptocrs/main.go

```diff
@@ -9,8 +9,10 @@ import (
 	"io/ioutil"
 	"log"
 	"os"
+	"path"
```

Comment:
What is this commit fixing? Or, what is G304 ?

Reply:
I can add a better commit msg
https://securego.io/docs/rules/g304.html